### DISCUSSION TITLE
Orchard take too much memory in the latest version

### DISCUSSION THREAD

**Discussion by hoanghai1212** [OrchardCMS/OrchardCore#9940] [Q&A]
I don't because of my computer or something but when I running Orchard by vs2019 it's show I using almost 1.5gb memory for only Orchard. I have deployed on aws ebs services using docker to run my Orchard, but when I working with Orchard cms about 15 - 20min it make my server return 502 Bad Gateway error. Anyone have same issue like me ?

**Comment by Skrypt** [comment]
If you are debugging it will take much more ram.

**Reply by hoanghai1212** [reply]
![image](https://user-images.githubusercontent.com/57449818/126469268-7bb002fa-3278-4c3f-b354-3243c84d534a.png)
I deploy release version but it still broken my server :(

**Comment by ns8482e** [comment]
I use NuGet feed, while working on my application it works and debugs well with VS2019.

To work with OrchardCore source, I use vs code.

**Comment by RockNHawk** [comment]
Use nuget package,it takes only 150~200MB memory.
